tornado/autoreload.py

```diff
@@ -135,9 +135,13 @@ def wait():
     to run the tests again after any source file changes (but see also
     the command-line interface in `main`)
     """
-    io_loop = ioloop.IOLoop()
-    start(io_loop)
-    io_loop.start()
+    # stop and clear the current io loop as we need a new one
+    ioloop.IOLoop().stop()
```

Comment:
Stopping the running IOLoop isn't needed (or appropriate) here. I think all that's needed is to change `start(io_loop)` to `io_loop.add_callback(start)`.

Reply:
Ah yeah of course, thanks! :slightly_smiling_face: